src/main/scala/edu/ie3/simona/agent/grid/congestion/CongestedComponents.scala

```diff
@@ -108,7 +108,7 @@ object CongestedComponents {
           val vMag =
             vNom * nodeRes(transformer.lvNodeUuid).getvMag.toSquants.toEach
 
-          val power = sqrt(3.0) * res.getiBMag.toSquants * vMag
+          val power = res.getiBMag.toSquants * sqrt(3.0) * vMag
```

Comment:
It triggers my inner monk that the sqrt can't stand at the start of the formula. Just a personal opinion, so you could leave it as it is, but I would place the sqrt 3 at the end in these cases.

Reply:
A added a new method in `edu.ie3.util.scala.quantities.QuantityUtil`. With this new extension, we can keep the code as it was before.